docs/technical/setup-local-dev-env.md

```diff
@@ -0,0 +1,53 @@
+# Setting up local development environment
+
+Prerequisite Technology Installs
```

Comment:
Just a minor thing, but it feels like if these sections were H2 headers (##), it would make the doc a bit easier to navigate

Reply:
I've updated #21 to address the headers issue. 

Again, this PR comment was also added when this PR was WIP. This file isn't part of the final changeset so I'm not going to make any changes to it in this PR.